resources/assets/less/bem/changelog-header.less

```diff
@@ -0,0 +1,58 @@
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.changelog-header {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-end;
+
+  height: 250px;
+
+  background-size: cover;
+  background-position: center;
+
+  .at2x-simple('/images/headers/changelog.jpg');
```

Comment:
then just use `background-image` because otherwise it'll use the nonexistent `@2x` file on certain setups.